src/transformers/models/whisper/convert_openai_to_hf.py

```diff
@@ -43,6 +54,22 @@
     "medium": "https://openaipublic.azureedge.net/main/whisper/models/345ae4da62f9b3d59415adc60127b97c714f32e89e936602e85993674d08dcb1/medium.pt",
     "large": "https://openaipublic.azureedge.net/main/whisper/models/e4b87e7e0bf463eb8e6956e646f1e277e901512310def2c24bf0e11bd3c28e9a/large.pt",
     "large-v2": "https://openaipublic.azureedge.net/main/whisper/models/81f7c96c852ee8fc832187b0132e569d6c3065a3252ed18e56effd0b6a73e524/large-v2.pt",
+    "large-v3": "https://openaipublic.azureedge.net/main/whisper/models/e5b1a55b89c1367dacf97e3e19bfd829a01529dbfdeefa8caeb59b3f1b81dadb/large-v3.pt",
+}
+
+_ALIGNMENT_HEADS = {
```

Comment:
I would be more inclined to store the alignment heads as Python lists (rather than base64 encodings)! I think this will simplify a lot of the loading logic for you: https://gist.github.com/hollance/42e32852f24243b748ae6bc1f985b13a